docs/lang/articles/basic/field.md

```diff
@@ -3,100 +3,137 @@ sidebar_position: 3
 ---
 
 # Fields
+Taichi fields are used to store data.
+In general, fields are used as global data containers that can be read and written from both the Python scope and the Taichi scope.
 
-Fields are **global** variables provided by Taichi. **Global** indicates that fields can be read/written from both the Python scope and the Taichi scope. A field can be considered as a multi-dimensional array of elements, and it can be either **dense** or **sparse**. Similar to a NumPy `ndarray` object, a field has a data type and a shape. Moreover, an element of a field can be a scalar, a **vector**, a **matrix**, or a **struct**.
+A field has its own data type and shape and can be considered as a multi-dimensional array of elements.
+An element of a field can be a **scalar**, a **vector**, a **matrix**, or a **struct**.
+The sparsity of a field element is **dense** by default, but it can also be **sparse**, as detailed in [Sparse computation](/lang/articles/advanced/sparse).
 
-The term **field** is borrowed from mathematics and physics. If you
-have already known [scalar field](https://en.wikipedia.org/wiki/Scalar_field) (e.g., heat field) or vector field (e.g., [gravitational field](https://en.wikipedia.org/wiki/Gravitational_field)) in mathematics and physics, it will be straightforward to understand the fields in Taichi.
-
-To be noticed:
-* Fields are always accessed by indices.
-* Field values are initially zero.
-* Sparse fields are initially inactive.
-
-:::tip
-In earlier versions of Taichi, you could not allocate new fields after executing the first kernel. Since Taichi v0.8.0, you can use a new class `FieldsBuilder` for dynamic field allocation and destruction. For more details, please see [Field (advanced)](/lang/articles/advanced/layout).
+:::note
+The term **field** is borrowed from mathematics and physics.
+If you have already known [scalar field](https://en.wikipedia.org/wiki/Scalar_field) (e.g., heat field) or vector field (e.g., [gravitational field](https://en.wikipedia.org/wiki/Gravitational_field)) in mathematics and physics,
+it will be straightforward to understand the fields in Taichi.
 :::
 
 ## Scalar fields
+We start introducing fields from this very basic type, the elements of scalar fields are simply scalars.
+* A 0D scalar field is a single scalar.
+* A 1D scalar field is a 1D linear array.
```

Comment:
I am wondering what the `linear` here indicates? Does it mean it has a `dense` memory layout?